src/index.css

```diff
@@ -85,3 +87,11 @@ circle {
   stroke-linecap: round;
   stroke-linejoin: round;
 }
+
+.inputWithAt::before {
+  content: '@';
+  position: absolute;
+  left: 0.5rem;
+  top: 0.5rem;
+  color: var(--formal-accent);
```

Comment:
Colour will be `formal-accent` by default, I think, not sure if it will work for `::before`, but lets try

Reply:
There is no need for it anymore. Thanks